libs/teller-api/src/teller-api.ts

```diff
@@ -153,15 +153,10 @@ export class TellerApi {
                 headers: {
                     Accept: 'application/json',
                 },
-            })
-
-            this.api.interceptors.request.use((config) => {
-                // Add the access_token to the auth object
-                config.auth = {
-                    username: 'ACCESS_TOKEN',
-                    password: accessToken,
-                }
-                return config
+                auth: {
+                    username: accessToken,
+                    password: '',
+                },
```

Comment:
there might be something funky going on here with the API singleton. The accessToken needs to be updated if you're getting accounts for different providers since the accessToken is unique to an enrollment/AccountConnection